engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeMethodNode.java

```diff
@@ -572,10 +584,10 @@ Object doConvertDate(
       UnresolvedSymbol symbol,
       Object self,
       Object[] arguments,
-      @CachedLibrary(limit = "10") TypesLibrary types,
-      @CachedLibrary(limit = "10") InteropLibrary interop,
-      @CachedLibrary(limit = "10") WarningsLibrary warnings,
-      @Cached MethodResolverNode methodResolverNode) {
+      @Shared("typesLib") @CachedLibrary(limit = "10") TypesLibrary types,
+      @Shared("interop") @CachedLibrary(limit = "10") InteropLibrary interop,
+      @Shared("warnsLib") @CachedLibrary(limit = "10") WarningsLibrary warnings,
```

Comment:
Wouldn't it be better to keep the name of argument and string send to `@Shared` in sync? E.g. use `@Shared("warnings")`?

Reply:
Renamed in 7118056bb79aeb0a7e25790f7304e5ce25b34ab2.